views/index.erb

```diff
@@ -1,12 +1,48 @@
 <% if @profile %>
-  <h3><%= @profile["first_name"] %>'s Amazing Profile</h3>
+  <h2 id="profile_title"><%= @profile["first_name"] %>'s Amazing Profile</h2>
 
   <!-- TODO- Perhaps re-jig this to fit your new design / layout -->
+  <div id="navdiv">
+    <div id="nav_button">
+        <h3>Navigate</h3>
+          <img class = "arrow" id="downarrow" src="images/downarrow.png" />
+          <img class = "arrow hidden" id="uparrow" src="images/uparrow.png" />
+    </div>
+    <nav id="navbar">
+      <ul>
+        <li><a href="#">About</a></li>
+        <li><a href="#ambition">Ambition</a></li>
+        <li><a href="#secondarycontent">General</a></li>
+        <li><a href="#projects">Projects</a></li>
+        <li><a href="#rationale">Rationale</a></li>
+        <li><a href="#problems">Problems</a></li>
+        <li><a href="#editbox">Edit</a></li>
+      </ul>
+    </nav>
+  </div>
+  <div id="maincontent">
+  <img id="profile-pic" src="images/me3.JPG" />
+  <img id="inksplat" src="images/inksplat.jpg" />
+
+  <a href="http://twitter.com/ctweedz"><img id="twitterlogo" src="images/twitterlogo.png" alt="twitter" /></a>
+  <a href="http://github.com/cltweedie"><img id="githublogo" src="images/github.png" alt="github" /></a>
+  <a href="http://codecademy.com/cltweedie"><img id="codecademylogo" src="images/codecademy.jpg" alt="codecademy" /></a>
   <% @profile.each do |field, value| %>
-    <div class="profile-section box">
-      <h4><%= field %></h4>
+    <% if (field == "codecademy") %>
```

Comment:
We tend to omit the parentheses around boolean expressions unless we need to influence the precedence. Take a look at the Ruby style guide for some general stylistic ideas: https://github.com/bbatsov/ruby-style-guide